airflow/bin/cli.py

```diff
@@ -257,6 +258,48 @@ def variables(args):
         print(msg)
 
 
+@cli.action_logging
+def pools(args):
+    """
+    set a batch of Airflow pools
+    :param args: filepath to airflow pools yaml file
+    :return:
+    """
+    with open(args.filepath, 'r') as stream:
+        try:
+            pools_config = yaml.load(stream)
+        except yaml.YAMLError as exc:
+            print(exc)
```

Comment:
You don't raise here, so if an error is hit here, we'll print the error, then get a `UnboundLocalException` on line 277 because pools_config won't be defined. 

You either need to exit execution, or give pools_config a default. 

Also, when printing exceptions and recovering, it's usually a good idea to give some context along with the exception. Especially because you're not formatting the traceback with the exception.

Reply:
sure

Reply:
+1, I didn't catch this.